api/nodes/serializers.py

```diff
@@ -526,9 +528,10 @@ def to_representation(self, obj):
         type_ = getattr(meta, 'type_', None)
         assert type_ is not None, 'Must define Meta.type_'
         relation_id_field = self.fields['id']
-        attribute = relation_id_field.get_attribute(obj)
+        attribute = obj.primary_institution._id if obj.primary_institution else None
         relationship = relation_id_field.to_representation(attribute)
-
+        if relationship == 'None':
```

Comment:
So, because it its a CharField (the attribute ` institution_id ` is), it get this:
```
def to_representation(self, value):
        return six.text_type(value)
```
will continue to investigate if it can be turned into ` null `